src/riot-rs-macros/tests/ui/config_conflicting_config_kind.stderr

```diff
@@ -0,0 +1,7 @@
+error: custom attribute panicked
+ --> tests/ui/config_conflicting_config_kind.rs:9:1
+  |
+9 | #[riot_rs::config(network, usb)]
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  |
+  = help: message: a separate function is required for `usb` configuration
```

Comment:
minor: I find this help message confusing. how about "only one configuration per function is supported."?